document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/service/DocumentGeneratorService.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.document.generator.core.service;
+
+import uk.gov.companieshouse.document.generator.core.service.models.DocumentRequest;
+import uk.gov.companieshouse.document.generator.core.service.models.DocumentResponse;
+
+public interface DocumentGeneratorService {
+
+    //TODO service interface created impl to be done as part of SFA 664
+    /**
+     * generate the document
+     *
+     * @param documentRequest
+     * @return
```

Comment:
Is there meant to be a return here?

Reply:
Yes, it returns a Document Response

Reply:
Can the Javadoc be updated to reflect that please?